### PR TITLE
net-nntp/nzbget: bump to 20.0_pre2108, improve init script

### PR DESCRIPTION
**Pull request by sbraz**
I didn't change anything to the ebuild. However, I noticed that the init script's stop command didn't work any more so I rewrote it to use nzbget's PID file.
@swegener can you test it and let me know if it works correctly for you?

### CODE REVIEW

File: net-nntp/nzbget/files/nzbget.initd [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
+command=/usr/bin/nzbget
+command_args="--configfile \"${NZBGET_CONFIGFILE}\" \
+  --daemon \
+  ${NZBGET_OPTS}"
+start_stop_daemon_args="--user \"${NZBGET_USER}\" \
+  --group \"${NZBGET_GROUP}\""
+pidfile=$(sed -n 's/^LockFile=//p' "${NZBGET_CONFIGFILE}")
+
+depend() {
+  need localmount net
```
Comment (SoapGentoo) [inline net-nntp/nzbget/files/nzbget.initd]: is this indentation right?

**Comment by swegener** [comment]
Do we want to keep 19.1 for stabilization? 14.1 is quite old.

For the init script I prefer

```shell
pidfile=/run/nzbget/nzbget.pid
command=/usr/bin/nzbget
command_args="--configfile \"${NZBGET_CONFIGFILE}\" --option LockFile=${pidfile} --daemon ${NZBGET_OPTS}"
start_stop_daemon_args="--user \"${NZBGET_USER}\" --group \"${NZBGET_GROUP}\""
```
I don't like the extra commnds to parse out values from configuration files. You need to handle if the value is unset or is changed during runtime etc.

**Comment by sbraz** [comment]
@swegener I didn't realise we could force the location of the PID file like this :)

**Comment by swegener** [comment]
I've pushed the commited. I've copied the init script to -r1 so that is doesn't apply to the stable ebuild.

**Pull request closed without merging**